cmd/policygen/main.go

```diff
@@ -27,29 +27,13 @@ import (
 
 var (
 	inputConfig = flag.String("input_config", "", "Path to the Policy Generator config.")
```

Comment:
With this simplified interface could we be consistent with the engine and do

--config_path
--state_path
--output_path

Also, I thought state is always in JSON? In the future can state path be a GCS path as well where we can recursively parse all states within the bucket? I think that will make it easy to support the engine generated configs.

Reply:
Sounds good. Given this PR is already pretty large. I'll do that in a followup PR.